LICENSE.md

```diff
@@ -0,0 +1,41 @@
+# Creative Commons CC0 1.0 Universal
```

Comment:
better to do plain text license that github can pick up because it has standard text

Reply:
I think it will do fine. I'll make sure to check when merged.